bootstrap.js

```diff
@@ -15,6 +15,7 @@ import { GridBox } from "./class/track/GridBox.js";
 
 const COMPILED = false;
 export const DEBUG = !COMPILED;
+window.debugMode = DEBUG;
```

Comment:
Wouldn't it be the same to use DEBUG?
The only benefit is you can change it through the console but then again you could just make a keyboard event later on to toggle it

Reply:
I prefer a global variable over a keyboard shortcut, but it doesn't matter too much.